### ISSUE TITLE
Accessibility: MAS20B: Link Purpose: Repeated "Overview" links are displayed in the page

### ISSUE THREAD

**Issue by wiproaccess** [wiproaccess/vscode-website#5]
#A11yMAS
#MAS20B
#Wipro
#vscode-website

**Testing Environment:**
OS-RS1
OS Name Microsoft -Windows 10 Pro
Browser-Edge
Edge Version-Microsoft Edge 38.14393.461.0

**Repro Steps:**
1.Open https://code.visualstudio.com/docs
2.Observe the "Overview" link is being repeated  at LHS pane under each main content  below the header of the page.[Screenshot]

**Actual Result:**
Repeated "Overview" link is observed at LHS pane under each main content below the header of the page.

**Expected Result:**
Link names should not be repeated. All Links in the page should have Unique Name.

**User Impact:**
Visually Impaired users relying on screen readers gets confused if links with same name getting announced multiple times
![repeated overview links](https://cloud.githubusercontent.com/assets/24825441/21672363/1d767c82-d348-11e6-940d-fb669de3ee02.png)

**Comment by chrisdias** [comment]
fixed with https://github.com/Microsoft/vscode-website/commit/2cdc3c7d171346d193c68bb0074f2be7c085555c

**Comment by waderyan** [comment]
Fixed
